docker-build.sh

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+##############################################################
+# Docker-based build script for StarRocks
+# This script uses the official StarRocks dev-env Docker image
+# to build the project in a consistent Ubuntu environment.
+#
+# Usage:
+#    ./docker-build.sh --help
```

Comment:
rename to `build-in-docker`? `docker-build.sh` is really close to `docker build` which produces the container image, not build the project in docker env.